web/controllers/vote_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule SignDict.VoteController do
+  use SignDict.Web, :controller
+
+  plug Guardian.Plug.EnsureAuthenticated, handler: SignDict.GuardianErrorHandler
+
+  alias SignDict.Vote
+  alias SignDict.Video
+
+  def create(conn, %{"video_id" => video_id}) do
+    IO.puts "creating"
```

Comment:
This line should be removed. Unless you have any special reason for this being here? 😄 Looks like an older debugging statement.